examples/third_party_examples/ReliableGPT_Getting_Started.ipynb

```diff
@@ -0,0 +1,122 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4vt1KClmFRP8"
+      },
+      "outputs": [],
+      "source": [
+        "#@title # Environment Set-Up\n",
+        "!pip install openai reliableGPT"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title # Basic Usage\n",
+        "#@markdown ## ReliableGPT handles\n",
+        "#@markdown * OpenAI APIError, OpenAI Timeout, OpenAI Rate Limit Errors, OpenAI ServiceUnavailableError / Overloaded\n",
+        "#@markdown * Context Window Errors\n",
+        "\n",
+        "import openai\n",
+        "from reliablegpt import reliableGPT\n",
+        "openai.api_key = \"\" #@param {type:\"string\"}\n",
+        "# ReliableGPT wraps around the chat completion endpoint, and safely handles common errors\n",
+        "user_email = \"krrish@berri.ai\" #@param {type:\"string\"}\n",
+        "openai.ChatCompletion.create = reliableGPT(openai.ChatCompletion.create, user_email= user_email)\n",
+        "\n",
+        "completion = openai.ChatCompletion.create(model=\"gpt-3.5-turbo\", messages=[{\"role\": \"user\", \"content\": \"Hello world\"}])\n",
+        "print(completion.choices[0].message.content)"
+      ],
+      "metadata": {
+        "id": "tjJi6ebbKI03",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "48ef02af-a7cb-4939-cfbd-6e7bde590f15"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Hello! How can I assist you today?\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title ## 🤯 Running 200 GPT-4 OpenAI Calls at once!\n",
```

Comment:
why does this say 200 calls? I see `range(20)` along with a messages that contains a super long string thanks to the `*400`.